lib/init-app.js

```diff
@@ -27,6 +27,7 @@ export default function({ rootPath, serverIoCFactory }) {
   const iocContainer = serverIoCFactory({
     baseConfigPath: baseConfigPath,
     configPath: configPath,
+    manifestPath
```

Comment:
The same, let's rename `manifestPath`  into `assetsManifestPath` to provide more explicit meaning.